src/main/java/com/github/jmchilton/blend4j/galaxy/HistoriesClient.java

```diff
@@ -37,6 +38,20 @@
   HistoryExport exportHistory(String historyId);
   
   /**
+   * Deletes the given History from Galaxy (this will not purge).
+   * @param historyId  The id of the History to delete.
+   * @return A {@link ClientResponse} for this request.
+   */
+  ClientResponse deleteHistoryResponse(String historyId);
```

Comment:
You can just use the `ClientResponse`.  It's proably the easiest way since there's no need to have access the the response data normally.

However, for using the API, you'll then have to check the HTTP status code of the response object.  As in:

```java
if (!ClientResponse.Status.OK.equals(response.getClientResponseStatus())) {
    // everything is okay
} else {
    // throw exception
}